apps/mull-api/src/main.ts

```diff
@@ -13,6 +13,7 @@ async function bootstrap() {
   const globalPrefix = 'api';
   app.enableCors();
   app.setGlobalPrefix(globalPrefix);
+  app.useGlobalPipes(new ValidationPipe());
```

Comment:
What does this do?

Reply:
This is to enable validation on all endpoints (including GraphQL). I followed instructions [here](https://docs.nestjs.com/techniques/validation) to get started and this works in conjunction with the class-validator package.